core/src/main/java/com/klaytn/caver/transaction/accessList/AccessList.java

```diff
@@ -0,0 +1,95 @@
+package com.klaytn.caver.transaction.accessList;
+
+import org.web3j.rlp.*;
+import org.web3j.utils.Numeric;
+
+import java.util.*;
+
+/**
+ * Represents an access list.
+ * AccessList is an EIP-2930 access list.
+ */
+public class AccessList extends ArrayList<AccessTuple> {
+    /**
+     * Constructor.
+     */
+    public AccessList() {}
+
+    /**
+     * Creates an AccessList instance from given list of AccessTuple.
+     * @param items A list of AccessTuple.
+     */
+    public AccessList(List<AccessTuple> items) {
+        this.addAll(items);
+    }
+
+    /**
+     * Creates an Access list.
+     *
+     * @param items
+     * @return
+     */
+    public static AccessList create(List<AccessTuple> items) {
+        AccessList accessList = new AccessList(items);
+        return accessList;
+    }
+
+    /**
+     * Returns a decoded access list.
+     *
+     * @param accessListRlpList List of RlpType to decode.
+     * @return AccessList
+     */
+    public static AccessList decode(RlpList accessListRlpList) {
+        List<RlpType> accessTupleRlps = accessListRlpList.getValues();
+        AccessList accessList = new AccessList();
+        for (int i =0 ; i < accessTupleRlps.size(); i++) {
+            accessList.add(AccessTuple.decode((RlpList) accessTupleRlps.get(i)));
+        }
+        return accessList;
+    }
+
+    /**
+     * Returns a decoded access list.
+     *
+     * @param rlpEncoded Rlp encoded string of access list.
+     * @return
+     */
+    public static AccessList decode(String rlpEncoded) {
+        return AccessList.decode(Numeric.hexStringToByteArray(rlpEncoded));
+    }
+
+    /**
+     * Returns a decoded access list.
+     *
+     * @param rlpEncoded Rlp encoded byte array of access list.
+     * @return
+     */
+    public static AccessList decode(byte[] rlpEncoded) {
+        RlpList rlpList = RlpDecoder.decode(rlpEncoded);
+        return AccessList.decode(((RlpList) rlpList.getValues().get(0)));
+    }
+
+    /**
+     * Returns the RLP-encoded string of this accessList.
+     *
+     * @return RlpList
+     */
+    public RlpList toRlpList() {
+        List<RlpType> rlpTypeList = new ArrayList<>();
+        for (AccessTuple accessTuple : this) {
+            rlpTypeList.add(accessTuple.toRlpList());
+        }
+        return new RlpList(rlpTypeList);
+    }
+
+    /**
+     * Returns an encoded access tuple.
+     *
+     * @return byte[]
+     */
+    public byte[] encodeToBytes() {
```

Comment:
Don't you need to implement `equals`?

Reply:
@jimni1222 
Yeah. It works fine.
`AccessTuple` already implemented `equals` and AccessList is a ArrayList<AccessTuple>, so it uses `AccessTuple.equals` when it need to be checked equal or not.

Below is the "Call Stack" when trying to compare AccessList objects by using `assertEquals`. You can check it by running [test](https://github.com/klaytn/caver-java/pull/291/commits/48d492a64bfbb25b7b93f524dd9d6ebb46352d45).
------
`AccessTuple.equals`
`AbstractList.equals`
...
`assertEquals`
`AccessList$decodeTest `
------